### ISSUE TITLE
Samsung Galaxy S21 Ultra

### ISSUE THREAD

**Issue by Videonas** [darktable-org/darktable#14969]
I couldn't open my DNG photos in Darktable. There seems to be no support for Samsung Galaxy S21 Ultra DNG's in the software (not sure if this applies to all Samsung Galaxy DNG's images). I would appreciate it if support for my phone could dome to Darktable. Therefore, I uploaded 5 images that I created to raw.pxls.us. I wasn't able to double-check that the uploads were successful. No image turned up in the repository overview immediately after upload. I'm guessing that there's some delay between upload and showing up in the overviews. In case uploads weren't successful: I'm terribly sorry, I could try again if I'm notified about this. In case you need more or other photos taken by my Samsung Galaxy S21 Ultra, I could provide those as well.

Hopefully, you received my photos and this request well and find the time and energy to implement support for my phone in Darktable.

Thank you very much in advance.

**Comment by kmilos** [comment]
Most likely duplicate of https://github.com/darktable-org/rawspeed/issues/403

**Comment by Nilvus** [comment]
You didn't precise on which OS you use darktable. Maybe mostly also a duplicate of: #11847

**Comment by Videonas** [comment]
Hey,

I tried to open my images in Darktable on Windows 10, TuxedoOS 2 and Fedora 38 and I got the same behaviour on all 3. 

The lighttable tab of Darktable 4.4.1 showed a black and white placeholder for my images. In Darktable 4.4.2, I get an actual preview of my .DNG images in the lighttable tab. In the darkroom tab, though, I get the notification that Darktable isn't able to open the images, that I have to check if my camera model is supported.

My apologies if this issue is a duplicate of another.
